### DISCUSSION TITLE
Enabling CBT support by default with vSphere Templates

### DISCUSSION THREAD

**Discussion by khessert** [okd-project/okd#1201] [General]
**Describe the bug**

PVCs fail to mount with error "CnsFault error: VSLM task failed" if CBT is not enabled on the vSphere VM
https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/1416

**Version**

IPI 4.9.0-0.okd-2022-01-14-230113 and 4.9.0-0.okd-2022-02-12-140851
vsphere_cpi_version: v1.22.4
vsphere_csi_version: v2.4.1
vCenter Server 7.0 Update 3a

**How reproducible**

100%

**Log bundle**

Will upload if requested

**Recommendation**

Is it possible to enable CBT by default in the vSphere Template by adding this line to the vmx file
ctkEnabled = "TRUE"

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

**Reply by khessert** [reply]
The cluster can be installed without any issues. We get an error as soon as PVs provisioned via VMware CSI are used. Which logs do you need in particular? Do you need all logs via "oc adm must-gather"?

**Reply by jcpowermac** [reply]
we didn't come to a conclusion on this yet. let me ask again

**Comment by LorbusChris** [comment]
https://github.com/openshift/installer/pull/5845
